VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/services/video-call.service.ts

```diff
@@ -44,7 +44,7 @@ export class VideoCallService {
     private onParticipantUpdatedSubject = new Subject<ParticipantUpdated>();
     private onConferenceUpdatedSubject = new Subject<ConferenceUpdated>();
     private onParticipantCreatedSubject = new Subject<ParticipantUpdated>();
-    private onParticipantDeletedSubject = new Subject<ParticipantUpdated>();
+    private onParticipantDeletedSubject = new Subject<string>();
```

Comment:
To keep in line with what the updated and created does, use a DTO

```suggestion
    private onParticipantDeletedSubject = new Subject<ParticipantDeleted>();
```